src/auth/components/finalizeLogin.tsx

```diff
@@ -29,9 +29,9 @@ const FinalizeLogin = (): JSX.Element | null => {
       successCallback={() => {
         redirect(savedRedirectUrl);
       }}
-      errorCallback={(e: Error) => {
+      errorCallback={() => {
         redirect(savedRedirectUrl);
-        console.error(e);
+        // console.error(e);
```

Comment:
Hmm, I thought the rule would only prevent `console.log`s. Retaining `console.error`s for development use is still useful, though. Maybe there could be a generic error handling function we can call for these error cases instead that checks for the environment and is exempt from the linter rule?

Reply:
That's a good idea. I like it more than just allow console.errors since those are not supposed to be used in prod env imo.

Reply:
The new function seems good, though I envisioned that it'd replace these existing `console.error` calls directly (that are currently just commented out rather than outright removed). I do like that logging of API errors is now automatic, it's just that the error objects now lose some context because the explanations added in some of those calls are no longer printed. There's also at least one non-API related error that no longer gets noted at all (the one about invalid favourites read from the localstorage).

Reply:
Yeah, seems good now.